Dfe.Academies.External.Web/Pages/YourApplications.cshtml

```diff
@@ -1,44 +1,52 @@
 ﻿@page
+@using Dfe.Academies.External.Web.Models
 @model Dfe.Academies.External.Web.Pages.HomeModel
 @{
     ViewData["Title"] = "Apply to become an academy - Your applications";
 }
 
 <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Your applications</h1>
+    
+    @{  
 
-	<div class="govuk-!-margin-6"></div>
+       if (Model.ExistingApplications != null)
```

Comment:
In my partials, I've checked the count on the list<>. I think this would be better, similar to below:-
Model.ValidationErrorMessages.Count > 0